### DISCUSSION TITLE
Requests: Checkboxes to Delete Multiple Requests

### DISCUSSION THREAD

**Discussion by war59312** [sct/overseerr#3215] [Feature Requests]
Currently you can only delete a "Request" 1 at a time and have to confirm.

Bit of a pain. Be nice if I could select all, select more than 1, etc. Think checkboxes would solve it.

And option to disable "confirm".

**Comment by tomatoinaction** [comment]
This is like the same pain as it is in plex to organize the watchlist.
I would also love that the "watched" status is getting synced

**Comment by sct** [comment]
> Currently you can only delete a "Request" 1 at a time and have to confirm.
> 
> 
> 
> Bit of a pain. Be nice if I could select all, select more than 1, etc. Think checkboxes would solve it.
> 
> 
> 
> And option to disable "confirm".

We do not encourage deleting requests.

**Comment by war59312** [comment]
Why is that? I only want them still list up until the point the request is fulfilled.

If there was a "fulfilled" status then sure I would have no need to delete if "fulfilled" would "archive" the request off the main screen and only show if filtered on.

**Reply by sct** [reply]
You can change the filter on the requests page to only show pending requests? (This won't change your home page but it should help with your main request list?)

As to why; it's because Overseerr uses request data for a lot of internal features. Things like quotas or tracking users' request history in general. We have chosen, by design, to keep request data as much as we can. Deleting is reserved for situations when things are, for example, requested by mistake.

**Comment by war59312** [comment]
Got you.

Then yes I guess the request would be, to be able to archive the requests so they no longer show on the home page and on /requests (would have a special archive filter to see them).

Then would have a nice home screen not filled with requests already fulfilled.
